api/openapi.yaml

```diff
@@ -500,3 +519,29 @@ components:
           example: "this is an error message"
       required:
         - message
+
+    Timestamp:
```

Comment:
Not in scope for this PR, but I think this is something we should add to all of our responses.

Reply:
removing timestamp for 0.6.0 as discussed

Reply:
TODO: add ticket for adding timestamp to endpoints

Reply:
Ticket created